docs/config/auth0.md

```diff
@@ -122,4 +122,4 @@ function() {
 
 Where `$ADD_USER_METADATA` is the [build output](../../apps/auth0-rules/build/add-user-metadata.js) of `@asap-hub/auth0-rules`.
 
-**You'll need to force the import of a specific `got` version. On the Auth0 editor, where you find `require('got')` replace by `require('got@$VERSION')`, where \$VERSION is the version that `yarn why got` shows for `@asap-hub/auth0-rules`.**
+**You'll need to force the import of a specific imports for example `got` version. On the Auth0 editor, where you find `require('got')` replace by `require('got@$VERSION')`, where \$VERSION is the version that `yarn why got` shows for `@asap-hub/auth0-rules`.**
```

Comment:
idk what happened here but this is not a sentence :joy: 
Do we have another external dependency other than `got`? Was that the intention behind this?

Reply:
We got additional dependencies using ??

Reply:
oh. some kind of babel runtime. can probably tweak babel config to inline all of that, then we can use `??` as normal again

Reply:
Is it really worth it? I would prefer if the code generated didn't require manual changes to use map instead of some imported one.